.github/workflows/mkdocs-master.yml

```diff
@@ -7,23 +7,25 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
       - name: set dev version
         working-directory: ./java
         run: echo "DEV_VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -Ev 'Download|INFO|WARNING')" >> $GITHUB_ENV
 
-      - uses: actions/setup-python@v2
+      - uses: actions/setup-python@v5
         with:
-          python-version: '3.8'
-      - run: pip3 install pip==22.0.3
+          python-version: "3.10"
 
       - name: install deps
         working-directory: ./python
         run: cp ../README.md . && pip3 install -e .[python,dev,docs]
 
+      - name: use dev mike
```

Comment:
I don't remember why we are pinning mike==1.1.2 in setup.py but then installing the latest dev version in the workflow. But it might be more reliable to instead pin mike to 2.0.0 in setup.py and not install mike from dev. Since a bug may leak out in dev causing issues for us.

Reply:
Ok, I think the issue currently is that pyparsing is not compatible with great-expectations but as long as regular user don't install docs I don't think it is an issue as you say. Will pin mike 2.0